docs/reference/health/health.asciidoc

```diff
@@ -356,6 +356,14 @@ watermark threshold>>.
     (Optional, array of strings) If corrupted repositories have been detected in the system, the names of up to ten of
     them are displayed in this field. If no corrupted repositories are found, this detail is omitted.
 
+`unknown_repositories`::
+    (Optional, int) The number of repositories on the system that have been determined to be unknown. If there are
```

Comment:
Maybe here we should stress the fact that this is a node specific issue. For example, 

> The number of repositories that have been determined to be unknown by at least one node in the system.

What do you think?

Reply:
Good point, thanks for the suggestion :)